### ISSUE TITLE
Set the levels?

### ISSUE THREAD

**Issue by zwardnz** [kvittingseerup/IsoformSwitchAnalyzeR#19]
Hi Kristoffer,
So I have a design_matix, eg:
[for.kristoffer.xlsx](https://github.com/kvittingseerup/IsoformSwitchAnalyzeR/files/2563497/for.kristoffer.xlsx)

I think that I re-set the levels by doing:
design_matrix2$condition<- factor(design_matrix2$condition, levels = c("pre","post"))

But after having a look at the SwitchAnalyzed data (after I run analyzeORF) it looks like the conditions are the opposite way around.
Sorry thi isn't an issue with IsoformSwitchAnalyze but was hoping you could help so that my analysis is correct.

eg head((SwitchListAnalyzed, 3)

```
               iso_ref          gene_ref        isoform_id           gene_id condition_1 condition_2 gene_name gene_overall_mean gene_value_1
 isoComp_00008178 geneComp_00000529 ENST00000262113.8 ENSG00000036448.9        post         pre      <NA>          467.5494     401.4723
 isoComp_00008180 geneComp_00000529 ENST00000518203.5 ENSG00000036448.9        post         pre      <NA>          467.5494     401.4723
 isoComp_00008181 geneComp_00000529 ENST00000518513.1 ENSG00000036448.9        post         pre      <NA>          467.5494     401.4723
```

**Comment by kvittingseerup** [comment]
Thanks for reaching out.

That is unfortunately a problem with IsoformSwitchAnalyzeR where factors are currently not propagated. I have added it to the to do list but might be a while before that update is available.

In the meantime you can use the "comparisonsToMake" argument in importRdata() to fix the comparison order.

Hope this helps.
Cheers
Kristoffer

**Comment by kvittingseerup** [comment]
This was fixed in the 1.5.4 update
